public/jsx/startup.jsx

```diff
@@ -285,6 +279,15 @@ var TourItem = React.createClass({
 });
 
 var Tour = React.createClass({
+  componentDidMount: function() {
+    $("#tour").owlCarousel({ singleItem: true });
+  },
+  shouldComponentUpdate: function() {
+    return false;
+  },
+  componentWillUnmount: function() {
+    $("#tour").data('owlCarousel').destroy();
+  },
```

Comment:
`owlCarousel` operates outside of React, so it can be moved into these lifecycle methods instead of a `setTimeout`.